libr/bin/p/bin_pe.inc

```diff
@@ -420,7 +420,7 @@ static int haschr(const RBinFile* bf, ut16 dllCharacteristic) {
 	if (!bf) {
 		return false;
 	}
-	buf = r_buf_data (bf->buf, &sz);
+	buf = r_buf_data_cached (bf->buf, &sz);
```

Comment:
this should not use `r_buf_data` either.

indeed this function just reads few bytes from the buffer, so you can use `r_buf_read_at` (or `r_buf_read_le16_at`, etc.) to read the bytes and `r_buf_size` to get the size. There is no need to read the whole buffer.